tsconfig.json

```diff
@@ -1,12 +1,17 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "declaration": true,
+    "esModuleInterop": true,
     "module": "commonjs",
+    "noFallthroughCasesInSwitch": true,
+    "noImplicitReturns": true,
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "outDir": "./dist",
+    "sourceMap": true,
     "strict": true,
-    "declaration": true,
-    "outDir": "./dist"
+    "target": "es5"
   },
-  "files": [
-    "./index.ts"
-  ]
+  "include": ["test"],
```

Comment:
Relatively new to TypeScript—what is the difference between `include` and `files`?

`include`'s docs say that they "take a list of glob-like file patterns", but this looks like a folder. Is that okay?

Also, if tests pass without `include`, why do we need to include it?

Reply:
I originally included `test` since without it, TypeScript wasn't performing type-checking on the `test` folder.

I've since found that removing both `files` and `include` means that type-checking is performed on all `.ts` files in the project, so have done this.